CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add support for `LidSwitchStateChange` in `PowerBroadcastSetting`.
```

Comment:
Can you please move this under the existing "Added" header?

Reply:
Done